demisto_sdk/commands/common/files/yml_file.py

```diff
@@ -0,0 +1,13 @@
+from pydantic import Field, validator
+
+from demisto_sdk.commands.common.files.json_yml_file import JsonYmlFile
+from demisto_sdk.commands.common.handlers import DEFAULT_YAML_HANDLER as yaml
+from demisto_sdk.commands.common.handlers import YAML_Handler
+
+
+class YmlFile(JsonYmlFile):
```

Comment:
why isnt this in the files directory ?

Reply:
nice catch, just by mistake, ill fix it

Reply:
we got confused, it is indeed in the files directory